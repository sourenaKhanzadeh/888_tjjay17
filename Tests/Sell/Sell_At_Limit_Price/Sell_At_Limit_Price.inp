login
sell-standard
sell
Under twenty five
999.99
