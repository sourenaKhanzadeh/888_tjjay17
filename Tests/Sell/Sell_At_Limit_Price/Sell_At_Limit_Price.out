Welcome to the system
Enter session type
Enter operation
Enter the event title you would like to sell
Enter the price of each ticket
Enter the number of tickets for sale
Session terminated
